source/game/field/Object.hh

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "game/system/map/MapdataGeoObj.hh"
+
+namespace Field {
+
+class Object {
+public:
+    Object(const System::MapdataGeoObj &params);
+
```

Comment:
Missing d'tor? Oooh wait, to assert that this is an abstract base class, should this really be a pure virtual destructor?

Reply:
The destructor is not pure virtual. Rather, the `init` function will be.

I think a destructor is not required at the moment because the class is trivially destructible.